app/models/user.rb

```diff
@@ -18,6 +18,20 @@ class User < ActiveRecord::Base
 
   scope :uncategorized_since, -> (date) { where 'categorized IS NOT true AND DATE(created_at) >= ?', date }
 
+  def self.from_auth0_user(auth0_user)
```

Comment:
maybe `create_from_auth0_user`. Don't mind a bit of extra space, at the benefit of the extra understandability.